pyscriptjs/src/plugins/python/py_markdown.py

```diff
@@ -16,7 +16,7 @@ def configure(self, config):
         console.log(f"configuration received: {config}")
 
     def afterStartup(self, interpreter):
-        console.log(f"interpreter received: {interpreter}")
+        console.log("interpreter received:", interpreter)
```

Comment:
Two reasons:
1. Indeed the main problem is that `str(interpreter)` was raising an error (what do we expect it to do?)
2. `console.log("string", object)` should be preferred to `console.log(f"string {object}")` whenever possible since the former allows the engine to generate a rich representation of `object` whereas the second usually says something useless like `[object Object]`